tdrs-backend/tdpservice/users/models.py

```diff
@@ -14,6 +15,19 @@ class User(AbstractUser):
     stt = models.ForeignKey(STT, on_delete=models.CASCADE, blank=True, null=True)
     region = models.ForeignKey(Region, on_delete=models.CASCADE, blank=True, null=True)
 
+    # Note this is different from the `is_active` that comes from AbstractUser.
+    # Django will totally prevent a user with is_active=True from authorizing.
+    # This field `inactive_account` helps facilitate the "Inactive Account" message
+    # we send user's on the client, (which is not practical with the `is_active` field).
+    inactive_account = models.BooleanField(
```

Comment:
It is a little confusing to have this in addition to the `is_active` field that comes from Django. Very open to an alternative name.

Reply:
While I'd like to clarify further, perhaps use of 'deactivated' throughout would resolve this? It was indeed confusing to see both is_active and inactive fields.